src/utils/sortGoods.ts

```diff
@@ -0,0 +1,28 @@
+import { SortType } from '../types/SortType';
+
+export function sortGoods(
+  goods: string[],
+  sortField: SortType = SortType.Default,
+  isReverse = false,
+): string[] {
+  const visibleGoods: string[] = [...goods];
```

Comment:
```suggestion
  const visibleGoods = [...goods];
```
You don't have to specify type every time, typescript understands what type it has